File: Python/Logic-1/caught_speeding.py
def caught_speeding(speed, is_birthday):
  if speed <= 60:
    return 0
  elif is_birthday and speed <= 65:
    return 0
  elif is_birthday and speed >= 66 and speed <= 85:
    return 1
  elif speed > 60 and speed <= 80:
    return 1
  return 2

File: Java/Logic-1/caughtSpeeding.java
public int caughtSpeeding(int speed, boolean isBirthday) {
  if(speed < 61 | (speed <66 && isBirthday)) return 0; 
  if((!isBirthday && speed >= 61 && speed <= 80) | (isBirthday && speed >= 66 && speed <= 85)) return 1;
  return 2;
}

